lib/in_parallel.rb

```diff
@@ -185,7 +185,7 @@ def self._execute_in_parallel(method_sym, obj = self, &block)
       ret_val                   = nil
       # Communicate the return value of the method or block
       read_result, write_result = IO.pipe
-      Dir.mkdir('tmp') unless Dir.exists? 'tmp'
+      Dir.mkdir('tmp') unless Dir.exist? 'tmp'
```

Comment:
I read through this and didn't want to make too many changes, but I did think about using `FileUtils.mkdir_p`.